scaffolds/java/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use this image as a base
+FROM niaquinto/gradle
+MAINTAINER @toch
+
+# In case someone loses the Dockerfile
+RUN rm -rf /etc/Dockerfile
+ADD Dockerfile /etc/Dockerfile
+
+RUN groupadd toch
+RUN useradd -s /bin/bash -m -d /usr/bin/app -g toch toch
+
+# Set your desired user as default
+USER toch
```

Comment:
Is there a way to use env vars rather than statically defining 'toch' in a dockerfile ? I suppose not